src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -62,7 +62,7 @@
 def schedule_organizations(timestamp: float | None = None, duration: int | None = None) -> None:
     if timestamp is None:
         # The time that the report was generated
-        timestamp = to_timestamp(floor_to_utc_day(timezone.now()))
+        timestamp = to_timestamp(timezone.now())
```

Comment:
do we need `to_timestamp`?
doesn't `timezone.now` already return a timestamp?

Reply:
`timezone.now()` returns a datetime object, not a timestamp. I might be able to remove it but I'll have to update every place we touch time...

Reply:
yeah this is a nightmare, do you think it's important?

Reply:
🤔 kinda wild to me that we even have a `to_timestamp` helper.

If you have a datetime object, then you could also just do 
```
from datetime import datetime

now = datetime.now()

timestamp = datetime.timestamp(now)
```

I think i would opt for this rather than our home grown one 🤔

Reply:
Why wouldn't we want to use it rather than manually doing the conversion ourselves?